src/language/mongoDBService.ts

```diff
@@ -40,6 +41,10 @@ import LINKS from '../utils/links';
 
 import DIAGNOSTIC_CODES from './diagnosticCodes';
 
+// Adopting dns result order changes with Node v18 and vscode 1.82.0-insider update.
+// Refs https://github.com/microsoft/vscode/issues/189805
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
should be safe to have it here, since vscode is doing the same .. is this process shared with other extensions and the main vscode process / renderer?

Reply:
I think it is not shared with the main process, otherwise, it would work without our fix.